Makefile.webos

```diff
@@ -97,7 +97,7 @@ HAVE_SHADERPIPELINE = 1
 HAVE_STB_FONT = 1
 HAVE_STB_IMAGE = 1
 HAVE_STB_VORBIS = 1
-HAVE_STDIN_CMD = 0
+HAVE_STDIN_CMD = 1

```

Comment:
Is this necessary? I don't think we are able to use that interface anyway, outside of debugging? (stdin/stdout of apps on webOS is unused/inaccessible when starting up normally from the UI)